### ISSUE TITLE
Specifying HVSC location crashes Sidplay 3.0 in Win10

### ISSUE THREAD

**Issue by ImreOlajos** [IngeHP/Sidplay3#3]
On Windows 10 when I start up Sidplay 3.0, I go to Settings -> HVSC, I click Browse to choose the location of my HVSC, I press OK - Sidplay 3.0 seems to load something, then it crashes. This is with HVSC v77.

**Comment by buxxNL** [comment]
Funny workaround for this bugs: It works when you add the path while actually playing a SID.

**Comment by ImreOlajos** [comment]
> Funny workaround for this bugs: It works when you add the path while actually playing a SID.

Yes, that does work. But when I exit Sidplay3 and re-open it, the HVSC path is yet again empty.

**Comment by buxxNL** [comment]
Even more workarounds: I think I eventually also closed Sidplay3 while playing a SID, then the path stuck. But the bug needs fixed.

**Comment by dhry** [comment]
Workaround: manually edit Sidplay3.ini (in same folder as app) and add the folder location manually. Example:

**[HVSC]**
Directory=J:\\!SIDMusic\HVSC\
ShowSTILEntry=Yes
ShowBug=Yes
ShowSTILglobal=Yes
STILAllSubtunes=No
STILAutoSize=No

**Comment by ImreOlajos** [comment]
> Workaround: manually edit Sidplay3.ini (in same folder as app) and add the folder location manually. 

I can confirm that this workaround works for me, too.
